modules/oadp-1-5-0-release-notes.adoc

```diff
@@ -0,0 +1,223 @@
+You can use the optional Velero server parameter `--item-block-worker-count`to run additional worker threads to process items in parallel. To enable this in OADP, set the `dpa.Spec.Configuration.Velero.ItemBlockWorkerCount` parameter to an integer value greater than zero.
+
+link:https://issues.redhat.com/browse/OADP-5635[OADP-5635]
+
+.OADP logs are now available in the JSON format
+
+With the of release {oadp-short} 1.5.0, the logs are now available in the JSON format. It helps to have pre-parsed data in their Elastic logs management system.
+
+link:https://issues.redhat.com/browse/OADP-3391[OADP-3391]
+
+.The `oc get dpa` command now displays Status
+
+With this release, the `oc get dpa` command now displays Status to improve user experience.
+
+link:https://issues.redhat.com/browse/OADP-1338[OADP-1338]
+
+[id="resolved-issues-1-5-0_{context}"]
+== Resolved issues
+
+.Containers now use `FallbackToLogsOnError` for `terminationMessagePolicy`
+
+With this release, for all the {oadp-first} Operator containers such as `operator-manager`, `velero`, `node-agent`, and `non-admin-controller`, the `terminationMessagePolicy` field can now set to the `FallbackToLogsOnError` value. This change ensures that if a container exits with an error and the termination message file is empty, {OCP-short} will use the last portion of the container's log output as the termination message.
+
+link:https://issues.redhat.com/browse/OADP-5183[OADP-5183]
+
+.Namespace admin can now access the application after restore
+
+Previously, the Namespace admin could not execute an application after the restore operation with the errors such as `exec operation is not allowed because the pod's security context exceeds your permissions`, `unable to validate against any security context constraint`, or `not usable by user or serviceaccount, provider restricted-v2`.
+
+With this update, this issue is now resolved and the namespace admin can access the application successfully after the restore.
+
+link:https://issues.redhat.com/browse/OADP-5611[OADP-5611]
+
+.Specifying status restoration at the individual resource instance level using the annotation is now possible
+
+Previously, status restoration was only configured at the resource type using the `restoreStatus` field in the Restore custom resource (CR). With this release, you can now specify the status restoration at the individual resource instance level using the following annotation:
+
+[source,terminal]
+----
+metadata:
+  annotations:
+    velero.io/restore-status: "true"
+----
+
+link:https://issues.redhat.com/browse/OADP-5968[OADP-5968]
+
+
+.Restore is now successful with `excludedClusterScopedResources`
+
+Previously, on performing the backup of an application with the `excludedClusterScopedResources` field set to `"storageclasses", "Namespace"` parameter, the backup was successful but the restore partially failed. With this update, the restore is successful.
```

Comment:
I do not understand -- "the `excludedClusterScopedResources` field set to `"storageclasses", "Namespace"` parameter" -- does this mean that the field was set to either storageclasse or  Namespace?? Also are these the values that go in the YAML? not sure about back ticks and double quotes too.